lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -400,7 +400,7 @@ class AwsCompileFunctions {
         // Try convert reservedConcurrency to integer
         const reservedConcurrency = Number(functionObject.reservedConcurrency);
 
-        if (_.isInteger(reservedConcurrency)) {
+        if (!isNaN(reservedConcurrency)) {
```

Comment:
Let's actually remove this condition (assign value unconditionally)

Reply:
@medikoo when I assign value unconditionally, I get a fail test case inde ```index.test.js```. Should I remove this test case?
This test case:

> 4) AwsCompileFunctions
       #compileFunctions()
         should throw an informative error message if non-integer reserved concurrency limit set on function:
     AssertionError: expected promise to be rejected with an error including 'You should use integer as reservedConcurrency value on function: new-service-dev-func' but it was fulfilled with [ 'func' ]
      at Proxy.<anonymous> (node_modules\chai-as-promised\lib\chai-as-promised.js:169:53)
      at Proxy.<anonymous> (node_modules\chai-as-promised\lib\chai-as-promised.js:47:29)
      at Proxy.methodWrapper (node_modules\chai\lib\chai\utils\addMethod.js:57:25)
      at Context.<anonymous> (lib\plugins\aws\package\compile\functions\index.test.js:2390:69)
      at processTicksAndRejections (internal/process/task_queues.js:76:11)

Reply:
Yes let's remove this test case